crm_timesheet/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+CRM Timesheet
+=============
+
+This module allows to generate timesheets from leads/opportunities and phone
+calls.
+
+
+Usage
+=====
+
+* In lead/opportunity forms you have Timesheet page.
+* In phone calls tree, you can add an analytic account, and if the call has
+  any duration, a timesheet activity will be created.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/117/8.0
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
+.. branch is "8.0" for example
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/hr-timesheet/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/hr-timesheet/issues/new?body=module:%20crm_timesheet%0Aversion:%208.0.1.0.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
version here is the Odoo version (8.0), as it is not expected that people remember to bump this version when changing manifest version